src/containers/Storage/StorageGroups/StorageGroups.tsx

```diff
@@ -123,8 +129,7 @@ function StorageGroups({data, tableSettings, visibleEntities, nodes, onShowAll}:
             header: tableColumnsNames[TableColumnsIds.Type],
             render: ({value, row}) => (
                 <>
-                    <Label>{value as string || '—'}</Label>
-                    {' '}
+                    <Label>{(value as string) || '—'}</Label>{' '}
```

Comment:
why?

Reply:
Fix made by prettier. I didn't paid enough attention, it looked better previously. Will fix it